generate/templates/templates/nodegit.js

```diff
@@ -1,3 +1,5 @@
+/* eslint no-unused-vars: "off" */
```

Comment:
Disable lint from the entire file imo. Perhaps we could have a post generate linter?